modules/mcuboot/hooks/nrf53_hooks.c

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
If you are planning to use custom image list in the serial recovery mode (`CONFIG_BOOT_MGMT_CUSTOM_IMG_LIST=y`) for fetch status of installation new image on the NET core the need to implement additionally `boot_img_install_stat_hook()`.